__tests__/app.test.js

```diff
@@ -465,6 +532,34 @@ describe("PATCH", () => {
   });
 });
 
+describe("DELETE", () => {
+  test("/api/comments/:comment_id 204: should delete and return a 204 (No content) indicating that it has successfully deleted the comment.", () => {
+    return request(app).delete("/api/comments/6").expect(204);
+  });
+
+  test("/api/comments/:comment_id 410: should reject and return a 410 (Gone) indicating that the comment doesn't even exist, either because it has already been deleted or the comment never existed in the parameter in the first place.", () => {
+    return request(app)
+      .delete("/api/comments/6768")
+      .expect(410)
+      .then(({ body }) => {
+        expect(body).toEqual({
+          message: "??? There isn't a comment stored in this parameter.",
+        });
+      });
+  });
+
+  test("/api/comments/:comment_id 410: should reject and return a 406 indicating that the parameter is not a nuber (ID).", () => {
+    return request(app)
+      .delete("/api/comments/HAVGVDG")
+      .expect(406)
+      .then(({ body }) => {
+        expect(body).toEqual({
+          message: "Unknown endpoints are not acceptable on this platform.",
+        });
+      });
```

Comment:
Please refactor your tests here. You would need to account for the following:

- Status 404, non existent ID, e.g 999
- Status 400, invalid ID, e.g "not-an-id"

Try to keep your test descriptions clear and concise 💡

Reply:
I did, from line 441 to line 473